pkg/proxy/ipvs/proxier.go

```diff
@@ -676,18 +683,19 @@ func (proxier *Proxier) syncProxyRules() {
 		"-j", "MASQUERADE",
 	}...)
 
-	if chain, ok := existingNATChains[KubeMarkMasqChain]; ok {
-		writeLine(proxier.natChains, chain)
-	} else {
-		writeLine(proxier.natChains, utiliptables.MakeChainLine(KubeMarkMasqChain))
-	}
 	// Install the kubernetes-specific masquerade mark rule. We use a whole chain for
 	// this so that it is easier to flush and change, for example if the mark
 	// value should ever change.
 	writeLine(proxier.natRules, []string{
 		"-A", string(KubeMarkMasqChain),
 		"-j", "MARK", "--set-xmark", proxier.masqueradeMark,
 	}...)
+
+	// Create chain KUBE-FIRE-WALL
+	if err := proxier.createKubeFireWallChain(existingNATChains, proxier.natChains); err != nil {
+		glog.Errorf("Failed to create KUBE-FIRE-WALL chain: %v", err)
```

Comment:
FWIW "firewall" is one word in English

Reply:
You mean `KUBE-FIREWALL` would be a better name?

Reply:
@Lion-Wei Yes, "KUBE-FIREWALL"